src/renderer/terminal/__tests__/atlasCoherence.test.ts

```
@@ -0,0 +1,46 @@
+  });
+
+  it('I2: allocating past maxPages never yields page >= maxPages', () => {
+    const max = 4;
+    const atlas = createAtlasModel(max);
+    for (let i = 0; i < 200; i++) atlas.rasterize(`g${i}`, 64, 64);
+    expect(atlas.pageCount).toBeLessThanOrEqual(max);
+  });
+
+  it('I3: two observers both see a wipe', () => {
+    const atlas = createAtlasModel(4);
+    let a = atlas.generation;
```

ESLint

[error] 'a' is never reassigned. Use 'const' instead.

(prefer-const)